rocAL/rocAL/include/loaders/image/node_fused_jpeg_crop_single_shard.h

```diff
@@ -43,10 +42,12 @@ class FusedJpegCropSingleShardNode: public Node
               unsigned num_attempts, std::vector<float> &random_area, std::vector<float> &random_aspect_ratio);
```

Comment:
@fiona-gladwin  remove extra space

Reply:
@LakshmiKumar23 may I know where exactly to remove the extra space?